regulations/templates/regulations/tree/preamble_intro.html

```diff
@@ -55,7 +55,7 @@ <h4 class="small-header">{{ meta.primary_agency }}</h4>
       <a class="citation internal" href="#supporting-docs">View supporting documents from the docket</a>
     </div>
     {% endif %}
-    <hr />
+    <hr class="preamble-divider" />
```

Comment:
Possible to use a `border-bottom` on `.view-supporting-docs` than `hr` in this case?